### ISSUE TITLE
White space control

### ISSUE THREAD

**Issue by rjhankison** [kalimatas/php-liquid#109]
Hello! We're a big fan of this package -- thank you very much for creating and maintaining!

One of our clients tried to [implement white space control.](https://shopify.github.io/liquid/basics/whitespace/) Example: 

    {%- assign my_variable = "tomato" -%}

However, when creating tags with a dash "-", we're experiencing a "Tag not properly terminated error". 
```
Liquid \ Exception \ ParseException
Tag {%- assign my_variable = "tomato" -%} was not properly terminated
```

We're using version 1.4.8. 

Is this something you're intending to support in future builds? 

Thank you very much in advance!
-Rich

**Comment by sanmai** [comment]
Hi, that's not what I have in plans yet, but if there's anyone to volunteer, reviews are on me.

**Comment by asacarter** [comment]
+1 for this feature.

**Comment by sanmai** [comment]
OK, this seems like doable. Give me some time.

**Comment by rjhankison** [comment]
Thanks very much, @sanmai! Really appreciate it. 

Is there an ETA on this? We're setting default template values for our DB in an upcoming release and will use the white space control if it's available.

**Comment by sanmai** [comment]
Guess it's only up to Travis CI to finish building #110.

**Comment by sanmai** [comment]
I'll reopen this as whitespace control is only implemented for tags, not variables.

**Comment by sanmai** [comment]
Fixed in #118
